### PR TITLE
:robot: Add fuzzdb.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/525 :rocket:

### CODE REVIEW

File: packages/fuzzdb.vm/fuzzdb.vm.nuspec [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>fuzzdb.vm</id>
+    <version>0.0.0.20230711</version>
+    <authors>fuzzdb-project</authors>
+    <description>FuzzDB was created to increase the likelihood of finding application security vulnerabilities through dynamic application security testing. It's the first and most comprehensive open dictionary of fault injection patterns, predictable resource locations, and regex for matching server responses.</description>
```
Comment (Ana06) [inline packages/fuzzdb.vm/fuzzdb.vm.nuspec:7]: [nitpick] I think a shorter description with less details would be better. From [our Wiki:](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#description)
> The description should be short and not include version specific details or other information that is likely to change in a future version.

Take this in mind even for future packages if you don't change it for this package. 😉

Comment (nos3curity) [reply packages/fuzzdb.vm/fuzzdb.vm.nuspec:7]: Agreed, thanks

**Comment by nos3curity** [comment]
@Menn1s, @day1player, @skollr34p3r please review

**Review by skollr34p3r — APPROVED** [review]
GTG

**Pull request merged**